stdlib/public/core/KeyPath.swift

```diff
@@ -59,6 +59,7 @@ public class AnyKeyPath: Hashable, _AppendKeyPath {
   ///   of this instance.
   @_effects(releasenone)
   final public func hash(into hasher: inout Hasher) {
+    hasher.combine(unsafeBitCast(type(of: self) as Any.Type, to: Int.self))
```

Comment:
This could be written `ObjectIdentifier(type(of: self)).hash(into: &hasher)`

Reply:
Good to know, thanks 👌